modules/ecs-ec2/otel_config.tftpl.yaml

```diff
@@ -48,16 +43,17 @@ processors:
           - set(attributes["aws_ecs_task_family"], attributes["aws.ecs.task.definition.family"])
           - set(attributes["image_id"], attributes["image.id"])
           - delete_key(attributes, "image.id")
-
+  batch:
```

Comment:
Lets use this defaults for now

`send_batch_size` 1024
`timeout` 1s
`send_batch_max_size` 2048

Reply:
While I'm ok with the OTEL defaults. I don't see any issue with the custom defaults.